sample_project/Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs

```diff
@@ -266,12 +270,11 @@ private void GroundedCheck()
 
         private void CameraRotation()
         {
+            //Don't multiply mouse input by Time.deltaTime;
```

Comment:
```suggestion
            // Don't multiply mouse input by Time.deltaTime
```

Reply:
This doesn't relate to the touch controls added in this PR and can be roped into the General Samples Code Cleanup issue.